src/SimpleVideoCutter/MainForm.cs

```diff
@@ -177,12 +183,12 @@ private void MainForm_Load(object? sender, EventArgs e)
                 }
             };
 
+            Globals.PlaceholderFiller = new PlaceholderFiller(videoCutterTimeline1);
```

Comment:
> This feels very wrong to me on many levels. 

I wholeheartedly agree. I already removed the functionality of proposing a relative output directory. However, before reading this comment I already refactored PlaceholderFiller, so it does not get videoCutterTimeLine1 in the constructor. My main goal was to remove the message "PlaceholderFiller may be null here", because I know that it cannot be null at the places where it is used.  I know that even the best solution to that has a drawback.
So, since it does not irk you to have FormMain own the instance of PlaceholderFiller (this feels slightly wrong, too), I will adjust my code accordingly.